nixos/lib/systemd-unit-options.nix

```
@@ -398,8 +398,18 @@ rec {
 
         enableStrictShellChecks = mkOption {
           type = types.bool;
-          description = "Enable running shellcheck on the generated scripts for this unit.";
-          # The default gets set in systemd-lib.nix because we don't have access to
+          description = ''
+            Enable running shellcheck on the generated scripts for this unit.
+            When enabled, scripts generated by the unit will be checked with
+            shellcheck and any errors or warnings will cause the build to fail.
+            This affects all scripts that have been created through the script,
+            reload, preStart, postStart, preStop and postStop options for
+            systemd services. This does not affect command lines passed directly
+            to ExecStart, ExecReload, ExecStartPre, ExecStartPost, ExecStop or
+            ExecStopPost.
```

```suggestion
            Enable running `shellcheck` on the generated scripts for this unit.
            When enabled, scripts generated by the unit will be checked with
            `shellcheck` and any errors or warnings will cause the build to fail.
            This affects all scripts that have been created through the `script`,
            `reload`, `preStart`, `postStart`, `preStop` and `postStop` options for
            systemd services. This does not affect command lines passed directly
            to `ExecStart`, `ExecReload`, `ExecStartPre`, `ExecStartPost`, `ExecStop` or
            `ExecStopPost`.
```
I recommend using backticks to distinguish technical terms from standard prose. Same goes for the other file.